DXRFixes/DeusEx/Classes/ActorDisplayWindow.uc

```diff
@@ -1,6 +1,30 @@
 class ActorDisplayWindow injects ActorDisplayWindow;
 
 var Font textfont;
+var bool bShowHidden;
+var bool bShowOnlyVisionImportant;
+
+function SetViewClass(Class<Actor> newViewClass)
+{
+    Super.SetViewClass(newViewClass);
+    bShowHidden = true;
+    bShowOnlyVisionImportant = false;
+}
+
+function ShowLOS(bool bShow)
+{
+    Super.ShowLOS(bShow);
+    bShowHidden = true;
+    bShowOnlyVisionImportant = false;
+}
+
+function ShowVisionImportant()
```

Comment:
Could instead create inviting actors of a new class to mark goals locations, with proper names, and the actors could be transient